api/v1beta1/cloudstackaffinitygroup_conversion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The Kubernetes Authors.
+Copyright 2023 The Kubernetes Authors.
```

Comment:
Would it be better to change it to `2022-2023`?

Reply:
I honest don't know. This maybe a legal question. I did not see 2021-2022 in vsphere .go files even we know it started before 2022.

Reply:
I looked at some of the vsphere files, and they contain the year that the file was created, not the year it was last updated.  Maybe we should leave the copyrights alone?

Reply:
I agree with you on this. 
But our repo has a `goheader` check that tries to make sure the copyright year is current year. Maybe we should not include it in our lint checking list. 

@davidjumani @jweite-amazon Do you have any opinion on this?

Reply:
Looking at other projects, it is the year created, not the current year. IMO the linter should be changed